torch/profiler/profiler.py

```diff
@@ -29,6 +32,11 @@
     "ExecutionGraphObserver",
 ]
 PROFILER_STEP_NAME = "ProfilerStep"
+_dynolog_handle = None
+
+logging.basicConfig(format="%(asctime)s:%(name)s:%(filename)s:%(lineno)s-%(levelname)s:%(message)s")
```

Comment:
Changing the logging global config is not ok.

Reply:
I couldn't find where the global config was defined, hence I did this. reverting it now. To confirm, is the following correct?
```
import logging
logger = logging.getLogger(__name__)
logger.setLevel(logging.INFO)
```